src/redux/configureStore.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable import/no-cycle */
 import { configureStore } from '@reduxjs/toolkit';
 import bookReducer from './books/books';
 import catReducer from './categories/categories';
```

Comment:
- [ ] (Optional) It would be nice to import `applyMiddleware`. This would help you use some handy Redux middleware, like a redux logger. The redux logger gives you details of the previous state and current state in the browser console. It would help you understand the inner workings of redux and the development process. 👍

Reply:
Let's leave it for now